sdcm/localhost.py

```diff
@@ -33,6 +35,18 @@ def __init__(self, user_prefix: Optional[str] = None, test_id: Optional[str] = N
     def rsyslog_port(self) -> Optional[int]:
         return ContainerManager.get_container_port(self, "rsyslog", RSYSLOG_PORT)
 
+    @property
+    def ldap_ports(self) -> Optional[dict]:
+        return {'ldap_port': ContainerManager.get_container_port(self, "ldap", LDAP_PORT),
+                'ldap_ssl_port': ContainerManager.get_container_port(self, "ldap", LDAP_SSL_PORT)}
+
+    @property
+    def ldap_config(self) -> Optional[dict]:
```

Comment:
how this configuration is related to the localhost class ? i'm not sure I'm following...

Reply:
the localhost configuration runs on the runner, and these configurations (or most of them) are sent to the container creation parameters...